reference/artifacts-with-artifactsrewrite/types/embedded-base64/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "Embedded Base64"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Rather than refer to a resource by a URI, you can embed the artifact's contents
+directly into the `"reference"` field in Base64.
+
+## Embedded Base64 Artifact in the UI
+
+The pipeline UI exposes the following fields for the embedded Base64 artifact:
+
+<table>
+  <thead>
+    <tr>
+      <th>Field</th>
+      <th>Explanation</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Account</strong></td>
+      <td>The "embedded-artifact" account.</td>
```

Comment:
Suggest changing the wording here to make it clear this is always just the string "embedded-artifact".  Maybe just:
Always "embedded-artifact".